Source/Krypton Toolkit/Main/Krypton.Toolkit.Suite.Extended.Messagebox/Classes/InternalKryptonMessageBoxExtended.cs

```diff
@@ -116,29 +117,17 @@ private void InitialiseComponent()
             _panelButtons.PanelBackStyle = PaletteBackStyle.PanelAlternate;
             _panelButtons.Size = new Size(156, 26);
             _panelButtons.TabIndex = 0;
-            //
+            /*//
             // _panelCheckBox
             //
             _panelCheckBox.Controls.Add(_optionalCheckBox);
-            _panelCheckBox.Dock = DockStyle.Left;
+            _panelCheckBox.Dock = DockStyle.Top;
```

Comment:
Should this have the possibility to be multi-line ?
Need to describe the restrictions on the "Amount of text" that goes int othis checkbox.

Reply:
I'm not sure if having two separate panels nested inside a 'footer' panel is going to work. Please could you take a look?

Reply:
I'm not sure how you are going to solve this otherwise. i.e.
Lot's of button (Max 4 ?) and some text (Max width dependent on Message contents ?)
- just use 2 panels and have the Checkbox AutoSize into the one above the buttons
- Or, 2 panels side by side (Via Flex table layout), with the buttons being the dominant size, and the checkbox being "Fill Rest".

Reply:
Or would it be more sensible to have the checkbox in a separate panel underneath the  buttons  panel, which may negate the long checkbox problem. The 4th button can be taken out, I was testing to see if a dedicated button to copy the message text to the clipboard can be implemented.

Reply:
If you are gong to implement it underneath the buttons, then this will follow the same pattern as the expanding messagebox. SO that might solve 2 issues and have the same look'n'feel ;-)

Reply:
Umm... a expandable messagebox, sounds like a good idea. Wonder if there is a way to create a circular `KryptonButton` to mimic the expanding button?